src/main/java/io/confluent/connect/hdfs/DataWriter.java

```diff
@@ -210,13 +211,18 @@ public void run() {
                 connectorConfig.getString(HdfsSinkConnectorConfig.WRITER_LOGGING_SCHEMA_REGISTRY_CONFIG));
 
         writerLogProducer = new KafkaProducer<>(props);
+        writerLogSchema = org.apache.avro.SchemaBuilder.record("writerlog").fields()
```

Comment:
Is there a reason to extract schema into `DataWriter` instead of putting it into a static variable inside `TopicPartitionWriter` class? I understand that we want to reuse the same object for all instances of `TopicPartitionWriter` which are continuously created while KC is running but I think we do not need to leak the schema outside of it